src/service/apiEndpoints.ts

```diff
@@ -114,4 +114,13 @@ export const endpoints = {
     list: <Endpoint>{ url: '/company/list', method: 'GET' },
     viewCompany: (slug: string) => <Endpoint>{ url: '/company/' + slug, method: 'GET' },
   },
+  campaignDocument: {
+    listDocuments: <Endpoint>{ url: '/campaign-document-role', method: 'GET' },
```

Comment:
Can you send me the PR where we have the API for endpoint `/campaign-document-role`

Reply:
On the other hand we have backend for `/campaign-file`

Reply:
I guess I should delete the role from [here]( https://github.com/podkrepi-bg/api/pull/149) too